apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.html.ts

```diff
@@ -395,23 +398,64 @@ export function LineitemsInputs(lineitems: Lineitems) {
     `.toString();
   }
   return html`
-    ${lineitems.map(
-      (lineitem) => html`
-        <div class="form-check">
-          <label class="form-check-label">
-            <input
-              class="form-check-input"
-              type="radio"
-              name="lineitem_id"
-              value="${lineitem.id}"
-              required
-            />
-            <span title="${lineitem.id}">${lineitem.label}</span>
-          </label>
-        </div>
-      `,
-    )}
+    <table class="table w-auto">
+      <tr>
+        <th>Assignment</th>
+        <th>Linked PrairieLearn assessment</th>
+      </tr>
+      ${lineitems.map(
+        (lineitem) => html`
+          <tr>
+            <td>
+              <div class="form-check">
+                <label class="form-check-label">
+                  <input
+                    class="form-check-input"
+                    type="radio"
+                    name="lineitem_id"
+                    value="${lineitem.id}"
+                    required
+                  />
+                  <span title="${lineitem.id}">${lineitem.label}</span>
+                </label>
+              </div>
+            </td>
+            <td>
+              ${lineitem.id in lti13AssessmentsByLineItemIdUrl
+                ? AssessmentLink({
+                    assessment:
+                      assessmentsById[lti13AssessmentsByLineItemIdUrl[lineitem.id].assessment_id],
+                    urlPrefix,
+                    spacerHtml: html``,
+                  })
+                : ''}
+            </td>
+          </tr>
+        `,
+      )}
+    </table>
     <button name="__action" value="link_assessment" class="btn btn-primary">Link assignment</button>
     ${disclaimer}
   `.toString();
 }
+
+function AssessmentLink({
+  assessment,
+  urlPrefix,
+  spacerHtml,
+}: {
+  assessment: AssessmentRow;
+  urlPrefix: string;
+  spacerHtml: HtmlSafeString | undefined;
+}) {
+  return html`
+    <a href="${urlPrefix}/assessment/${assessment.id}/"
+      ><span class="badge color-${assessment.assessment_set.color}">${assessment.label}</span></a
+    >
+    ${spacerHtml}
```

Comment:
Two notes:

- This `spacerHtml` thing is strange. You probably want two separate components: `AssessmentBadge` and `AssessmentLink`. Lucky for you, the former already exists at `apps/prairielearn/src/components/AssessmentBadge.html.ts`!
- For accessibility reasons, the badge should not be a link. See the first bullet point on this issue: https://github.com/PrairieLearn/PrairieLearn/pull/10490

Reply:
Good points, thanks. These are refactored.